lib/cib/cib_ops.c

```diff
@@ -731,7 +731,7 @@ cib_process_create(const char *op, int options, const char *section, xmlNode * r
                                   answer);
     }
 
-    failed = create_xml_node(NULL, XML_TAG_FAILED);
+    failed = create_xml_node(NULL, PCMK__XE_FAILED);
```

Comment:
another "throwaway" constant -- just one failure info XML node that needs a name